modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentItemLocalServiceImpl.java

```diff
@@ -583,18 +580,20 @@ private void _validate(
 				commerceInventoryWarehouseId);
 
 		if (commerceShipmentItem != null) {
-			availableQuantity =
-				availableQuantity + commerceShipmentItem.getQuantity();
+			availableQuantity = availableQuantity.subtract(
```

Comment:
why subtract?

Reply:
typo, thanks for that 👍 
I'll fix it asap